### PR TITLE
Add a new stylesheet to the project

### PR DESCRIPTION
**Pull request by StanleyAnyas**

### CODE REVIEW

**Review by Succynice — COMMENTED** [review]
@StanleyAnyas Pls kindly resolve conflicts, to enable a seamless merge

**Review by StanleyAnyas — COMMENTED** [review]
The id in the div is used to style the header to the middle

File: index.html [hunk 1]
```diff
@@ -12,30 +13,33 @@
...
-          sed, accusantium dolorum a est distinctio non itaque ipsam excepturi
-          harum nihil iure? Eum perspiciatis, animi ut ipsa facilis fugiat
-          vitae.
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Molestias sed,
+          accusantium dolorum a est distinctio non itaque ipsam excepturi harum
+          nihil iure? Eum perspiciatis, animi ut ipsa facilis fugiat vitae.
         </p>
       </section>
-      
-      <!-- Second section with header and paragraph -->
```
Comment (Succynice) [inline index.html:25]: @StanleyAnyas I think u hv deleted this comment by error

Comment (Succynice) [reply index.html:25]: Attend to this boss

Comment (StanleyAnyas) [reply index.html:25]: What do i have to do

Comment (Succynice) [reply index.html:25]: copy the comment and paste it right back into the codebase where it's meant to be

**Comment by StanleyAnyas** [comment]
You can check it now

File: index.html [hunk 2]
```diff
@@ -12,30 +13,34 @@
     <!-- This is the main content of the page -->
     <main>
       <!-- First section with header and paragraph -->
-      <section class="section1">
-        <h1>Testing Header One</h1>
+      <section>
```
Comment (Chiemella) [inline index.html:16]: Nice one

**Review by Chiemella — APPROVED** [review]

**Pull request merged**